WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -2687,7 +2695,30 @@ function WeakAuras.AddOption(id, data)
           end
           WeakAuras.ResetMoverSizer();
         end,
-        args = flattenRegionOptions(regionOption, id);
+        args = flattenRegionOptions(regionOption, id),
+        handler = {
```

Comment:
How does that work? I've never seen that.

Reply:
Every field in an AceConfig widget (except, irritatingly, for image) accepts a string value, in which case AceConfig will search down the options tree for a handler object, and call the relevant method. So in this case, on the collapse execute button, when clicked it calls `handler:CollapsedFunc(info, ...`.

I did it this way because I couldn’t see any other option to avoid RemoveFuncs from removing the function when multi-selecting.

Reply:
So how does that work for the other places where we use a func, such as e.g. expand code editor? Those do work without requiring a handler?